starter-code/functions-and-arrays.js

```diff
@@ -63,7 +84,17 @@ console.log(averageLength);
 
 // Unique Arrays
 function uniquifyArray (array) {
-
+  var copy = [];
```

Comment:
Al final habéis pillado lo de indexOf ¿no? No esta mal que penséis out of the box, pero pensad que en ocasiones el rendimiento sera muy importante. Buen trabajo!!